Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,53 @@
 import React from 'react';
 import './App.css';
 
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className='calculator-container'>
+      <div className='calc-top'>
+        <CalculatorDisplay text='0' />
+      </div>
+      <div className='buttons-all'>
+        <div className='buttons-left'>
+          <ActionButton btnType='action large clear' text='clear' />
+          <div className='buttons-numbered'>
+
+            <div className='row1'>
```

Comment:
Try to make the elements be all together and use flex wrap to make it more dynamic